IndicoV2/Submissions/SubmissionsClient.cs

```diff
@@ -25,47 +26,70 @@ public SubmissionsClient(IndicoClient indicoClient)
             _strawberryShakeClient = indicoClient.IndicoStrawberryShakeClient;
         }
 
-       
-        public Task<IEnumerable<int>> CreateAsync(int workflowId, IEnumerable<Stream> streams,
-            CancellationToken cancellationToken = default) =>
-            _legacy.CreateAsync(workflowId, streams, cancellationToken);
+        public async Task<IEnumerable<int>> CreateAsync(int workflowId, IEnumerable<Stream> streams, CancellationToken cancellationToken = default)
```

Comment:
does making a method `async` warrant an Obsolete tag and duplicate method? Methods both end up calling async function downstream and hence have same return type so I'm assuming the answer is no